source/includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs

```diff
@@ -0,0 +1,274 @@
+
+        // end-geoshape-search
+
+        return result;
+    }
+
+    public static List<Guitar> GeoWithinSearch()
+    {
+        // start-geowithin-search
+        GeoJsonPolygon<GeoJson2DGeographicCoordinates> searchArea = new(new(new(new GeoJson2DGeographicCoordinates[]
+        {
+             new(-74.3994140625, 40.5305017757),
+             new(-74.7290039063, 40.5805846641),
+             new(-74.7729492188, 40.9467136651),
+             new(-74.0698242188, 41.1290213475),
+             new(-73.65234375, 40.9964840144),
+             new(-72.6416015625, 40.9467136651),
+             new(-72.3559570313, 40.7971774152),
+             new(-74.3994140625, 40.5305017757),
+        })));
+
+        var result = guitarsCollection.Aggregate()
+            .Search(Builders<Guitar>.Search.GeoWithin(g => g.InStockLocation, searchArea))
+            .ToList();
+        // end-geowithin-search
+
+        return result;
+    }
+
+    public static List<Guitar> MoreLikeThisSearch()
+    {
+        // start-morelikethis-search
+        var searchDocument = new GuitarSearch()
+        {
+            Description = "Classic guitar",
+        };
+
+        var result = guitarsCollection.Aggregate()
+            .Search(Builders<Guitar>.Search.MoreLikeThis(searchDocument))
+            .ToList();
+        // end-morelikethis-search
+
+        return result;
+    }
+
+    public static List<Guitar> NearSearch()
+    {
+        // start-near-search
+        var result = guitarsCollection.Aggregate()
+            .Search(Builders<Guitar>.Search.Near(g => g.Rating, 9, 1))
+            .ToList();
+        // end-near-search
+
+        return result;
+    }
+
+    public static List<Guitar> PhraseSearch()
+    {
+        // start-phrase-search
+        var result = guitarsCollection.Aggregate()
+            .Search(Builders<Guitar>.Search.Phrase(g => g.Description, "classic guitars"))
+            .ToList();
+        // end-phrase-search
+
+        return result;
+    }
+
+    public static List<Guitar> PhraseMultipleSearch()
+    {
+        // start-multiphrase-search
+        var result = guitarsCollection.Aggregate()
+            .Search(Builders<Guitar>.Search.Phrase(g => g.Description, new List<string>() { "classic guitars", "quality guitars" }))
+            .ToList();
+        // end-multiphrase-search
+
+        return result;
+    }
+
+    public static List<Guitar> QueryStringSearch()
+    {
+        // start-querystring-search
+        var result = guitarsCollection.Aggregate()
+            .Search(Builders<Guitar>.Search.QueryString(g => g.Description, "(classic OR quality) AND NOT custom"))
+            .ToList();
+        // end-querystring-search
+
+        return result;
+    }
+
+    public static List<Guitar> RangeSearch()
+    {
+        // start-range-search
+        var result = guitarsCollection.Aggregate()
+            .Search(Builders<Guitar>.Search.Range(g => r.EstablishedYear, SearchRangeBuilder.Gt(1980).Lt(2020)))
```

Comment:
Correct me if I'm wrong, but this no longer works. You changed the initial parameter, but still call `r.EstablishedYear`. Shouldn't it be `g.EstablishedYear`?

Reply:
Oops. Got too hasty with my find and replace. Fixed